### PR TITLE
Fix ci failures

### PR DESCRIPTION
**Pull request by keelerm84**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ workflows:
               parameters:
                   php-version: ["8.0", "8.1"]
                   composer-dependencies: ["lowest", "highest"]
-      - test-on-windows
```
Comment (kinyoklion) [inline .circleci/config.yml:14]: Would we re-introduce this?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # LaunchDarkly OpenFeature provider for the Server-Side SDK for PHP
 
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/launchdarkly/openfeature-php-server/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/launchdarkly/openfeature-php-server/tree/main)
+[![Packagist](https://img.shields.io/packagist/v/launchdarkly/openfeature-server.svg?style=flat-square)](https://packagist.org/packages/launchdarkly/openfeature-server)
+[![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://launchdarkly.github.io/openfeature-php-server)
```
Comment (kinyoklion) [inline README.md:5]: This is a 404 currently, is that right?

File: .circleci/config.yml (hunk 1 above)
Comment (keelerm84) [reply .circleci/config.yml:14]: Once we address the problem upstream, yes, I think we would want to re-enable this.

File: README.md (hunk 2 above)
Comment (keelerm84) [reply README.md:5]: Yes. Once I publish the first release, we should be publishing some generated docs there.

Comment (kinyoklion) [reply README.md:5]: (Guess so, looks like no docs publish is in the branch.)

**Review by kinyoklion — APPROVED** [review]

**Pull request merged**